coaster-nn/src/frameworks/native/mod.rs

```diff
@@ -83,6 +87,7 @@ impl<T> NN<T> for Backend<Native>
     type CPOOL = helper::PoolingConfig;
     // type CACTI = helper::ActivationConfig;
     type CDROP = helper::DropoutConfig;
+    type RC = helper::RnnConfig;
```

Comment:
can we rename this to `CRNN`? with 2 letter acronyms we are slowly creating a very cryptic code base to other contributors.